### PR TITLE
update allowlist to include operator version 0.68.0

### PR DESCRIPTION
**Pull request by PaurushGarg**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by bryan-aguilar — COMMENTED** [review]
Please review this again to see if other tags need to be updated for `v0.68.0` mirroring. For example the target allocator.

**Review by vasireddy99 — APPROVED** [review]
LGTM!

**Comment by PaurushGarg** [comment]
> Please review this again to see if other tags need to be updated for `v0.68.0` mirroring. For example the target allocator.

Thanks. Updated

File: tools/release/adot-operator-images-mirror/config.yaml [hunk 1]
```diff
@@ -36,12 +38,14 @@ sourceRepos:
       - 1.18.0
       - 1.19.1
       - 1.20.2
+      - 1.22.1
```
Comment (bryan-aguilar) [inline tools/release/adot-operator-images-mirror/config.yaml:41]: Do we need to mirror the versions in between this? @erichsueh3 can you help with this question.

**Review by bryan-aguilar — CHANGES_REQUESTED** [review]

File: tools/release/adot-operator-images-mirror/config.yaml (hunk 1 above)
Comment (bryan-aguilar) [reply tools/release/adot-operator-images-mirror/config.yaml:41]: For reference `1.22.0`, and `1.21.0` are not in the mirror list.

**Review by bryan-aguilar — APPROVED** [review]

File: tools/release/adot-operator-images-mirror/config.yaml (hunk 1 above)
Comment (erichsueh3) [reply tools/release/adot-operator-images-mirror/config.yaml:41]: Nope, we only need to mirror the versions that are used with the Operator versions in the EKS add-on

**Review by erichsueh3 — APPROVED** [review]

**Pull request merged**